geolocation-API/PositionOptions.https.html

```diff
@@ -43,71 +43,50 @@
 
   try {
     geo.getCurrentPosition(
-        t86.step_func(function(pos) {
-          assert_unreached('A success callback was invoked unexpectedly');
-        }),
-        t86.step_func(function(err) {
+        t86.unreached_func('A success callback was invoked unexpectedly'),
+        t86.step_func_done(function(err) {
           assert_equals(err.code, err.TIMEOUT);
-          t86.done();
         }),
         {timeout: 0, maximumAge: 0}
     );
   } catch(e) {
-    t86.step(function() {
-      assert_unreached('An exception was thrown unexpectedly: ' + e.message);
-    });
+    t86.unreached_func('An exception was thrown unexpectedly: ' + e.message);
```

Comment:
Hmm you're right. I guess we could do `t86.unreached_func(...)()` but that'll be obscure. I'll leave it as it was.